coala_quickstart/Strings.py

```diff
@@ -33,3 +33,9 @@
              "all its contents but exclude your .git directory and all "
              ".o files. To do this, simply give `src/` for the first "
              "question and `.git/**,**/*.o` for the second question.")
+
+bear_help = ("A coala bear is a plugin that contains the actual algorithm. "
+             "It have its own native code or use an external linter. It "
+             "may be language specific or language independent. This "
+             "makes coala completely modularized and extensible. "
+             "Currently there are 78 bears for 36 different languages.")
```

Comment:
how to make stuff hard to maintain: you mention the number of bears and languages ;p usually i avoid this by saying: over 75 bears and over 35 languages :)